packages/gamut/src/GridForm/GridFormButtons/index.tsx

```diff
@@ -47,12 +48,21 @@ const buttonMap = {
 export const GridFormButtons: React.FC<
   GridFormSubmitProps & CancelButtonProps
 > = ({ type = 'fill', ...props }) => {
```

Comment:
```suggestion
> = ({ type = 'fill', position, ...props }) => {
```

Can change to the above, and then just use position instead of props.position in the instances on line 51, 60 & 62